setup.py

```diff
@@ -1,37 +1,34 @@
+# -*- coding: utf-8 -*-
```

Comment:
Are these comments needed?
Removing them seems to make no difference for more.

Reply:
see https://docs.python.org/3/reference/lexical_analysis.html#encoding-declarations
They are not needed, and indeed added by my pre-commit config. I took them out.